src/components/ButtonTable/ButtonTable.jsx

```diff
@@ -0,0 +1,20 @@
+import { Typography } from "@mui/material";
+import { buttonDetailStyle } from "../../buttonDelivery.styles";
+import Button from "../Button/Button";
+
+const ButtonTable = ({ Icon, children }) => {
```

Comment:
A better name for this should be secondary button. What if we want to use this button in another place that is not the table?

Also it shouldn't contain the `endIcon`, this `endIcon` should be passed as prop

Reply:
Example:
<img width="828" alt="Screenshot 2022-12-06 at 10 10 22 p m" src="https://user-images.githubusercontent.com/3700448/206086260-09fb4414-da02-4a04-a7c3-e98696ec546d.png">